templates/etc/td-agent/td-agent.conf

```diff
@@ -20,6 +20,12 @@
   @type null
 </match>
 
+# Add a unique insertId to each log entry. This helps guarantee the order and
```

Comment:
`to each log entry that doesn't already have it`, right?

Reply:
Yup! Changed.